docs/index.md

```diff
@@ -20,6 +20,7 @@ benchmarks
 contributing
 release-notes/index
 references
+zarr-v3
```

Comment:
Are you sure that top-level is the right place for that?

If you want to advertise it more prominently, there are other ways.

Reply:
I don't know where else to put it, open to ideas!

Reply:
It’s a guide, so maybe into `tutorials/`?

Reply:
Makes sense, I don't mind it being under tutorials.  We'll tweet about it/advertise lots :)